include/inset_state.hpp

```diff
@@ -91,18 +110,20 @@ class InsetState
   void apply_smyth_craster_projection();
 
   // Calculate difference between initial area and current area
-  double area_drift() const;
+  double area_expansion_factor() const;
```

Comment:
This function has a side effect that it prints the area drift value. Preferably, this function should have only one task, and we can have another function that prints the area_drift info.

Reply:
I fixed this! Moved control output out too.